template/_package.json

```diff
@@ -17,13 +17,13 @@
     "precommit": "npm run lint"<% } %>
   },
   "dependencies": {
-    "cross-env": "^5.0.1",
-    "nuxt": "^1.4.2"<% if (server === 'express') { %>,
-    "express": "^4.15.3"<% } else if (server === 'koa') { %>,
-    "koa": "^2.3.0"<% } else if (server === 'hapi') { %>,
-    "hapi-nuxt": "^0.6.0"<% } else if (server === 'micro') { %>,
-    "micro": "^9.0.0",
-    "micro-route": "^2.4.0"<% } else if (server === 'feathers') { %>,
+    "cross-env": "^5.2.0",
+    "nuxt-edge": "latest"<% if (server === 'express') { %>,
```

Comment:
Why not using directly `"nuxt": "^2.0.0"`?

Reply:
Because 2.0 is not released yet, no npm package, maybe we can also release create-nuxt-app edge same as nuxt-edge.

Reply:
Could you create 2.0 branch for this pr and change the base ?